docs/02-app/01-building-your-application/07-configuring/15-content-security-policy.mdx

```diff
@@ -90,21 +88,19 @@ export function middleware(request) {
     block-all-mixed-content;
     upgrade-insecure-requests;
 `
+    .replace(/\s{2,}/g, ' ')
+    .trim() // Replace newline characters and spaces
 
   const requestHeaders = new Headers(request.headers)
   requestHeaders.set('x-nonce', nonce)
-  requestHeaders.set(
-    'Content-Security-Policy',
-    // Replace newline characters and spaces
-    cspHeader.replace(/\s{2,}/g, ' ').trim()
-  )
+  requestHeaders.set('Content-Security-Policy', cspHeader)
 
-  return NextResponse.next({
```

Comment:
Could we continue using NextResponse.next()? Also, we can remove these comments I believe.